### PR TITLE
Replayer: convert global slots to avoid to hashes in order to support devnet

### PR DESCRIPTION
**Pull request by dkijania**
Changed global_slots_to_avoid to state_hashed_to_avoid array in order to support other networks, like denvet. Before the change, it was not possible to add slots from other network as global slot to avoid on mainnet not necessarily should be skipped in devnet and vice versa

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

**Review by emberian — APPROVED** [review]

**Comment by dkijania** [comment]
!ci-build-me

File: src/app/replayer/replayer.ml [hunk 1]
```diff
@@ -1213,17 +1213,74 @@ let main ~input_file ~output_file_opt ~archive_uri ~set_nonces ~repair_nonces
...
+            ; "3NKAbgwCDsdW6m18FSj8mBE2SpdMd7gopc5NMqGHxkqqLykMZrCR"
+            ; "3NKFtTCSqVqKdbm6unoCiQWnKnozwYsiey5aPegV5xMR2MNR9kjZ"
+            ; "3NKEyaQdmFtYWx2swLjkcpBeQGsfo8riNH6Kbdr9myWUQrZkbqjV"
+            ; "3NK7jasQXuACjzJ5mBPNr13Y6Yt8vD6piLc1waFBU4Jte8WzEZ4h"
+            ; "3NLPbRNQi6JFbh7DW9ibPgRhUnExwGaMpiuGJdEQ6Na47kWcP3PW"
+            ; "3NK4EUku6d9fmU3P1t7NnoxsN4sg699JSskJZ4nJs1mWSHs1CL4v"
+            ; "3NLDE8xKUvKMiSpLSd9uvnQaRiXYW6MWw7UXpFD2wbq4WeKRAnKM"
+            ; "3NLZYyohgxGFE3hw6o6tfvXnZLoG6gyzr67WNGPFfpugHtoXaFGq"
+            ]
+            @ (*mainnet*)
```
Comment (mrmr1993) [inline src/app/replayer/replayer.ml:1250]: Don't append, just add a comment in the list. Appending is very inefficient.

**Review by mrmr1993 — CHANGES_REQUESTED** [review]

**Comment by dkijania** [comment]
!ci-build-me

**Review by mrmr1993 — APPROVED** [review]

**Comment by dkijania** [comment]
!ci-build-me

**Pull request merged**